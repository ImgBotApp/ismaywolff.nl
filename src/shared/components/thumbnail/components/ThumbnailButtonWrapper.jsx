import styled from 'styled-components'

const ThumbnailButtonWrapper = styled.div`
  bottom: 0;
  color: var(--color-white);
  padding: var(--size-small);
  position: absolute;
  right: 0;
`

export default ThumbnailButtonWrapper
