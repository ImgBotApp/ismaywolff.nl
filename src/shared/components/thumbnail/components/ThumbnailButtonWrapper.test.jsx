import 'jest-styled-components'
import React from 'react'
import { shallow } from 'enzyme'
import ThumbnailButtonWrapper from './ThumbnailButtonWrapper'

describe('<ThumbnailButtonWrapper />', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<ThumbnailButtonWrapper />)
    expect(wrapper).toMatchSnapshot()
  })
})
