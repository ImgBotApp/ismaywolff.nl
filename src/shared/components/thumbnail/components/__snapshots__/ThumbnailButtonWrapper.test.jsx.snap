// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ThumbnailButtonWrapper /> renders correctly 1`] = `
.c0 {
  bottom: 0;
  color: var(--color-white);
  padding: var(--size-small);
  position: absolute;
  right: 0;
}

<div
  className="c0"
/>
`;
